enterprise-suite/values.yaml

```diff
@@ -79,7 +71,7 @@ imageCredentials:
 createAlertManager: true
 
 # Comma separated list of alertmanager addresses (can be k8s local DNS service names)
-alertManagers: alertmanager:9093
+alertManagers: alertmanager.$KUBE_CONSOLE_DOMAIN:9093
```

Comment:
How does `$KUBE_CONSOLE_DOMAIN` get set when we kick off console locally?  Does this need to be set in the Makefile?

Reply:
No it's not evaluated here - it gets copied directly into `default.conf` as `$KUBE_CONSOLE_DOMAIN`. Which then gets replaced at start up.

Reply:
Right.  D'oh!

Reply:
I think the problem is ```            - --alertmanagers={{ .Values.alertManagers }}
``` over in prometheus-deployment